### PR TITLE
Add literature background to char-rnn

### PR DESCRIPTION
**Pull request by lilianabs**
@CarloLucibello I created this PR with the changes that correspond to this example.

Reference issue # [340](https://github.com/FluxML/model-zoo/issues/340)

### CODE REVIEW

File: text/char-rnn/char-rnn.jl [hunk 1]
```diff
@@ -13,44 +41,70 @@ using Parameters: @with_kw
...
     Ys = collect(partition(batchseq(chunk(text[2:end], args.nbatch), stop), args.seqlen))
 
     return Xs, Ys, N, alphabet
 end
 
-# Function to construct model
+# The function `getdata` performs the following tasks:
+
+# * Downloads the [All works of Shakespeare concatenated](https://cs.stanford.edu/people/karpathy/char-rnn/shakespeare_input.txt) 
+# data if not previously downloaded. This function loads the data as a vector of characters with the function `collect`.
```
Comment (ToucheSir) [inline text/char-rnn/char-rnn.jl:75]:
```suggestion
# * Downloads a dataset of [all of Shakespeare's works (concatenated)](https://cs.stanford.edu/people/karpathy/char-rnn/shakespeare_input.txt) 
# if not previously downloaded. This function loads the data as a vector of characters with the function `collect`.
```

**Comment by lilianabs** [comment]
Thank you @ToucheSir for the suggestions! I committed the changes.

**Review by ToucheSir — APPROVED** [review]
LGTM, thanks. @CarloLucibello mind having another look so we can merge this?

**Comment by CarloLucibello** [comment]
Looks good, thanks!

**Pull request merged**